examples/contrib/cifar10/main.py

```diff
@@ -260,7 +260,7 @@ def initialize(config):
 
 def log_metrics(logger, epoch, elapsed, tag, metrics):
     metrics_output = "\n".join([f"\t{k}: {v}" for k, v in metrics.items()])
-    logger.info(f"\nEpoch {epoch} - Evaluation time (seconds): {elapsed:.2f} - {tag} metrics:\n {metrics_output}")
+    logger.info(f"Epoch[{epoch}] - Evaluation time (seconds): {elapsed:.3f}\n - {tag} metrics:\n {metrics_output}")
```

Comment:
If evaluation time is long, isn't more understandable to have:
```suggestion
    minutes, seconds = divmod(elapsed, 60)
    logger.info(f"Epoch[{epoch}] - Evaluation time : {int(minutes)} min and {int(seconds)} s\n - {tag} metrics:\n {metrics_output}")
```

Reply:
For cifar10 it wont be that long. It is around 2-3 seconds for resnet18. But in any case `elapsed` refers to `engine.state.times[Events.COMPLETED]` and where the time is seconds no matter how much time it took.